src/js/tech/tech.js

```diff
@@ -163,6 +163,10 @@ class Tech extends Component {
    * @param {string} src The source string at the time of the source changing.
    */
   triggerSourceset(src) {
+    if (!this.options_.enableSourceset) {
```

Comment:
I don't know if we should do this here. It makes manually using `sourceset` after init a bit weird. I guess we won't expect users to call `setupSourcesetHandling_()` on their own, but I think it is something that I would do. This would require setting an option and calling that function.

Reply:
Yeah, we don't expect and not recommend people call private/suffixed methods like that, though, I'd be ok with removing this bit regardless.